src/main/java/de/unistuttgart/overworldbackend/controller/PlayerNPCStatisticController.java

```diff
@@ -0,0 +1,46 @@
+package de.unistuttgart.overworldbackend.controller;
+
+import de.unistuttgart.overworldbackend.data.*;
+import de.unistuttgart.overworldbackend.repositories.PlayerNPCStatisticRepository;
+import de.unistuttgart.overworldbackend.repositories.PlayerTaskStatisticRepository;
+import de.unistuttgart.overworldbackend.service.PlayerNPCStatisticService;
+import de.unistuttgart.overworldbackend.service.PlayerTaskStatisticService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import java.util.UUID;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@Tag(name = "Player Task Statistik", description = "Get and update dungeons from worlds")
+@RestController
+@Slf4j
+@RequestMapping("/lectures/{lectureId}/playerstatistics/{playerId}/player-npc-statistics")
```

Comment:
Should we perhaps change this route to 
```suggestion
@RequestMapping("/lectures/{lectureId}/statistics/player/{playerId}/npc-stats")
```
?

Reply:
We need to make this discussen on another time, now is not the time